deploy/template/prometheus-rule.yaml

```diff
@@ -95,7 +95,7 @@ spec:
         - alert: KeycloakAPIRequestDuration90PercThresholdExceeded
           annotations:
             message: >-
-              Less than 90% of the RH SSO API endpoints in namespace {{ $labels.namespace }} are taking longer than 1s for the last 5 minutes.
+              90% of the total requests are not served within 10 seconds for the last 5 minutes for the RH SSO API in the {{ $labels.namespace }} namespace
```

Comment:
Shouldn't this say 1 second, not 10 seconds?

Reply:
Good catch :+1: